postgresql/src/main/java/org/apache/zeppelin/postgresql/PostgreSqlInterpreter.java

```diff
@@ -275,6 +276,9 @@ private InterpreterResult executeSql(String sql) {
    * For %table response replace Tab and Newline characters from the content.
    */
   private String replaceReservedChars(boolean isTableResponseType, String str) {
+    if (str == null) {
```

Comment:
I believe that the current implementation (e.g. replace `null` by empty string) should apply for non-table responses as well. Do you see an use case where this could be a problem?